packages/packages.json

```diff
@@ -1,36 +1,36 @@
-    "contract/valory/gnosis_safe/0.1.0": "bafybeifbkitkori3pi2mtevm475xbnljfejpci7u7veziaptrj4hjbciju",
-    "contract/valory/offchain_aggregator/0.1.0": "bafybeifjmazj5ger33po34easdzytqdgi4yaieqaa2vfs535sbgpdwnuf4",
+    "contract/valory/gnosis_safe/0.1.0": "bafybeihht4j726a4swe4duqxohnxtctdghftgywurw55zwjjqsbqjwnacm",
     "protocol/valory/acn/1.1.0": "bafybeidetrrkvdgveu4ph5g6v53lbh7ardfspbkpstmjxctx647bzyosyy",
-    "skill/valory/abstract_round_abci/0.1.0": "bafybeigpz64qx22a4zzlcczmc5frhlecytrh5q3x77lwe3a5kee2qrcchm",
+    "skill/valory/abstract_round_abci/0.1.0": "bafybeiasu3lbnwqdgdzoaxrvbos5qgabxtxkhb52r3rosrp4s4c246467a",
     "connection/valory/p2p_libp2p_client/0.1.0": "bafybeigvayl4ykzqf6o6bw2irv7am3qvczjoeu7yjhzn27ajiwvcio3lxm",
     "contract/elcollectooorr/basket/0.1.0": "bafybeievb6an7m7wtfy4qwqurfgqsoczfzmz5f6zqbkbsi7jjxyp6tofc4",
     "contract/elcollectooorr/token_vault/0.1.0": "bafybeif5cgknn7qjbm7js7jhhfv7zs2cp2eywcfgzru3mvlieqqwl2ohuq",
-    "skill/valory/transaction_settlement_abci/0.1.0": "bafybeig3lpuertmg5wngcnherollnvakvqgeisy3a6f5tbmw4e5k5khnt4",
+    "skill/valory/transaction_settlement_abci/0.1.0": "bafybeia2gs36s2mz32yaznhlidiolhw5swkumiej4vgwywphppjvurldie",
     "contract/elcollectooorr/artblocks/0.1.0": "bafybeiebk7ldudazfdxhvmiaz7r7u4pyupmfqflskgtosb26yuehif3dju",
     "contract/elcollectooorr/artblocks_minter_filter/0.1.0": "bafybeicpdjm3xdbg2a7cshg63mutuyac2xl2hnmtrptyx7rx3i6fneuzam",
     "contract/elcollectooorr/artblocks_periphery/0.1.0": "bafybeifnfae5b643q3z6j2jwj6mvwwlpidseetsl6jtcjkaiheqphibvpy",
     "contract/valory/multisend/0.1.0": "bafybeig45676hbh4c3p3mujrrskxgxww4cxdyyginlg5rmmav6orv4gtya",
     "skill/elcollectooorr/fractionalize_deployment_abci/0.1.0": "bafybeifi3egpy2yfvj4isjtsg2y3had7e4ifn2o7v32dkjozkusfvxepti",
-    "skill/valory/registration_abci/0.1.0": "bafybeiajvfkffbdmggj3it7e4r6733lylxhbnb2a6tx3tmlgpy2fpxkuze",
-    "skill/valory/reset_pause_abci/0.1.0": "bafybeieo5m37rwoel7e6u3lrp2cei7hdyhklggsc6776vej6bp6gzua7ei",
-    "skill/valory/safe_deployment_abci/0.1.0": "bafybeifkxk6jkixnk4fymsw76zw2z2ofshmomqbxkp2kcwpur6dn7jb44i",
+    "skill/valory/registration_abci/0.1.0": "bafybeie7lxe5spcitc7pwfilapsu5wlictp4ibz7gbko5yg3nx6llcjisu",
+    "skill/valory/reset_pause_abci/0.1.0": "bafybeiamkbck2c35uc6h7c6sotmsobr7h6tc2vzskoxztlu7o6af6mcacm",
+    "skill/valory/safe_deployment_abci/0.1.0": "bafybeifgljjbc2x42xcynmirchrno7iwwq6ufnhpqr37ypeoqstfnjujli",
     "skill/elcollectooorr/elcollectooorr_abci/0.1.0": "bafybeieco6qygpwen2uzzf7verj7ee3dziwvlvo6msjqzs4jxsbycpjgza",
     "agent/elcollectooorr/elcollectooorr/0.1.0": "bafybeia4zn34dgz6lgaogyjuxpmml27dz7w2qwemq2kxngiebzdln6sroa",
     "contract/elcollectooorr/token_settings/0.1.0": "bafybeiakmc2uikfzls3kwwpnokd645ih74avadiylbev5gxh4t6qqt7z64",
+    "contract/valory/offchain_aggregator/0.1.0": "bafybeifpfs2xb6y7dc6eugxlzqkki6hbici7r2glwk5gc55j6pssmjfrp4",
```

Comment:
This is no longer needed

Reply:
(which you can also see from the fact that no skill depends on it - packages.json flattens the dependency tree)